### PR TITLE
Added a note about data transformers not being applied with inherit_data option set

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.3+ |
| Fixed tickets | #4332 |

### CODE REVIEW

**Comment by javiereguiluz** [comment]
I need help to fix this cross link: https://github.com/symfony/symfony-docs/pull/5023/files#diff-10d22571d2214eea79758829d3bbf069R22

File: cookbook/form/data_transformers.rst [hunk 1]
```diff
@@ -17,6 +17,11 @@ You could try to do this in your controller, but it's not the best solution.
 It would be better if this issue were automatically converted to an Issue object.
 This is where Data Transformers come into play.
 
+.. caution::
+
+    When a field has the :doc:`inherit_data </reference/forms/types/options/inherit_data>`
```
Comment (WouterJ) [inline cookbook/form/data_transformers.rst]: you can't link to included files. So either link to an inherit_data option of the Form element reference (create a new ref pointer) or remove interlinking here.

**Comment by javiereguiluz** [comment]
This PR seems to be finished, so we can tag it with the *finished* label. Thanks.

**Comment by weaverryan** [comment]
Looks good to me! Thanks!

**Pull request merged**